Sources/SwiftParserDiagnostics/ParseDiagnosticsGenerator.swift

```diff
@@ -236,7 +236,7 @@ public class ParseDiagnosticsGenerator: SyntaxAnyVisitor {
     exchangeTokens(
       unexpected: misplacedSpecifiers,
       unexpectedTokenCondition: { EffectSpecifier(token: $0) != nil },
-      correctTokens: [effectSpecifiers?.asyncSpecifier, effectSpecifiers?.throwsSpecifier],
+      correctTokens: [effectSpecifiers?.asyncSpecifier, effectSpecifiers?.throwsClause?.throwsSpecifier],
```

Comment:
This won’t produce a correct Fix-It for the following because it will only move `throws` in front of `->` but not `MyError`.

```swift
func foo() -> throws(MyError) Int
```